packages/schema/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@orama/schema",
```

Comment:
@orama/plugin-json-schema

Reply:
I was thinking to keep it generic so if in the future you would like to generate the schema from other sources, like yaml or toml, you don't have lots of packages to maintain